### DISCUSSION TITLE
Cronicle

### DISCUSSION THREAD

**Discussion by Richdark** [selfhst/icons#71] [New Icon Request]
Since there is just a clock ticking at the place where a logo would usually appear, I think there is nothing closer to the Cronicle icon than this. For example: https://www.myqnap.org/wp-content/uploads/cronicle-logo.png

Cronicle website: https://cronicle.net
GitHub repo: https://github.com/jhuckaby/Cronicle

**Comment by shollyethan** [comment]
Cronicle has been added - thanks!
